src/pybind/ceph_argparse.py

```diff
@@ -1316,7 +1316,7 @@ def run_in_thread(func, *args, **kwargs):
     if timeout == 0 or timeout == None:
         # python threading module will just get blocked if timeout is `None`,
         # otherwise it will keep polling until timeout or thread stops.
-        timeout = 2 ** 32
+        timeout = (1 << (32 - 1)) - 1
```

Comment:
Is there any difference in comparison to just `2**31 - 1`?
Is the `(32 - 1)` supposed to bring more attention on the integer's size in bits? If so, a comment could make this clearer.

Reply:
no.
sure.

Reply:
@rzarzynski  fixed and repushed. could you take another look?